protocol/contracts/beanstalk/silo/ConvertFacet.sol

```diff
@@ -74,37 +74,54 @@ contract ConvertFacet is ReentrancyGuard {
         nonReentrant
         returns (int96 toStem, uint256 fromAmount, uint256 toAmount, uint256 fromBdv, uint256 toBdv)
     {
-        address toToken; address fromToken; uint256 grownStalk; address account;
+        address toToken; address fromToken; uint256 grownStalk; address account; bool decreaseBDV;
+
+        /** @dev account and decreaseBDV are initialized at the start of LibConvert.convert()
+        * as address(0) and false respectively and remain that way if a convert is not anti-lambda-lambda
+        * If it is anti-lambda, account is the address of the account to update the deposit
+        * and decreaseBDV is true */
+        (toToken, fromToken, toAmount, fromAmount, account, decreaseBDV) = LibConvert.convert(convertData);
 
-        (toToken, fromToken, toAmount, fromAmount) = LibConvert.convert(convertData);
-        
         require(fromAmount > 0, "Convert: From amount is 0.");
 
-        LibSilo._mow(msg.sender, fromToken);
-        LibSilo._mow(msg.sender, toToken);
+        // replace account with msg.sender if no account is specified
+        if(account == address(0)) account = msg.sender;
+
+        LibSilo._mow(account, fromToken);
```

Comment:
given that we now have two converts where to and from token are the same, think we can add logic to check whether toToken != fromToken and omit the 2nd mow.

Reply:
fixed [here](https://github.com/BeanstalkFarms/Beanstalk/pull/736/commits/b40ac94815c889d4318f7bfe76039ef55e5f1069)